inst/qml/qml_components/RobustBayesianMetaAnalysisAdvanced.qml

```diff
@@ -0,0 +1,244 @@
+				{ label: qsTr("logOR"),				value: "logOr"}
+			]
+		}
+
+		Group
+		{
+			title: 		qsTr("MCMC")
+
+			IntegerField
+			{
+				name:			"advancedMcmcAdaptation"
+				label:			qsTr("Adaptation")
+				defaultValue:	500
+				min:			100
+				fieldWidth:		55 * preferencesModel.uiScale
+			}
+			IntegerField
+			{
+				name:			"advancedMcmcBurnin"
+				label:			qsTr("Burnin")
+				defaultValue:	2000
+				min:			100
+				fieldWidth:		55 * preferencesModel.uiScale
+			}
+			IntegerField
+			{
+				name:			"advancedMcmcSamples"
+				label:			qsTr("Samples")
+				defaultValue:	5000
+				min:			100
+				fieldWidth:		55 * preferencesModel.uiScale
+			}
+			IntegerField
+			{
+				name:			"advancedMcmcChains"
+				label:			qsTr("Chains")
+				defaultValue:	3
+				min:			1
+				fieldWidth:		55 * preferencesModel.uiScale
+			}
+			IntegerField
+			{
+				name:			"advancedMcmcThin"
+				label:			qsTr("Thin")
+				defaultValue:	1
+				min:			1
+				fieldWidth:		55 * preferencesModel.uiScale
+			}
+
+		}
+
+		SetSeed{}
+	}
+
+	
+	Group
+	{
+
+		CheckBox
+		{
+			label:			qsTr("Autofit")
+			name:			"autofit"
+			checked:		true
+
+			CheckBox
+			{
+				label: 				qsTr("R-hat")
+				name:				"advancedAutofitRHat"
+				checked:			true
+				childrenOnSameRow:	true
+
+				DoubleField
+				{
+					name:			"advancedAutofitRHatTarget"
+					defaultValue:	1.05
+					min:			1
+					inclusive:		JASP.None
+				}
+			}
+
+			CheckBox
+			{
+				label: 				qsTr("Effective sample size")
+				name:				"advancedAutofitEss"
+				checked:			true
+				childrenOnSameRow:	true
+
+				DoubleField
+				{
+					name:			"advancedAutofitEssTarget"
+					defaultValue:	500
+					min:			1
+					inclusive:		JASP.None
+				}
+			}
+
+			CheckBox
+			{
+				label: 				qsTr("MCMC error")
+				name:				"advancedAutofitMcmcError"
+				checked:			false
+				childrenOnSameRow:	true
+
+				DoubleField
+				{
+					name:			"advancedAutofitMcmcErrorTarget"
+					defaultValue:	0.001
+					min:			0
+					inclusive:		JASP.None
+				}
+			}
+
+			CheckBox
+			{
+				label: 				qsTr("MCMC error / SD")
+				name:				"advancedAutofitMcmcErrorSd"
+				checked:			false
+				childrenOnSameRow:	true
+
+				DoubleField
+				{
+					name:			"advancedAutofitMcmcErrorSdTarget"
+					defaultValue:	0.01
+					min:			0
+					inclusive:		JASP.None
+				}
+			}
+
+			CheckBox
+			{
+				label: 				qsTr("Maximum fitting time")
+				name:				"advancedAutofitMaximumFittingTime"
+				checked:			false
+				childrenOnSameRow:	true
+
+				Group
+				{
+					Row
+					{
+						IntegerField
+						{
+							name:			"advancedAutofitMaximumFittingTimeTarget"
+							defaultValue:	1
+							min:			0
+						}
+
+						DropDown
+						{
+							name:			"advancedAutofitMaximumFittingTimeTargetUnit"
+							values:
+							[
+								{ label: qsTr("hours"),				value: "hours"},
+								{ label: qsTr("minutes"),			value: "mins"},
+								{ label: qsTr("seconds"),			value: "secs"}
+							]
+						}
```

Comment:
I am curious, how do you measure the time?

Reply:
`runjags` R package does that